app/templates/views/templates/template.html

```diff
@@ -56,18 +56,31 @@ <h2 class="font-body-lg">
           {{ template._template.updated_at|format_delta }}
         </time>
       </h2>
-      <a class="usa-link margin-right-3" href="{{ url_for('.view_template_versions', service_id=current_service.id, template_id=template.id) }}">See previous versions</a>
-    {% endif %}
-    {% if current_user.has_permissions('manage_templates') and user_has_template_permission %}
-      {% if not template._template.archived %}
-          <a class="usa-link margin-right-3" href="{{ url_for('.delete_service_template', service_id=current_service.id, template_id=template.id) }}">Delete this template</a>
-      {% endif %}
-      {% if not template._template.redact_personalisation %}
-          <a class="usa-link" href="{{ url_for('.confirm_redact_template', service_id=current_service.id, template_id=template.id) }}">Hide personalization after sending</a>
-      {% else %}
+      <h3 class="margin-bottom-0">More actions</h3>
+      <ul class="usa-list usa-list--unstyled">
+        <li>
+          <a class="usa-link margin-right-3"
+          href="{{ url_for('.view_template_versions', service_id=current_service.id, template_id=template.id) }}">See previous
+          versions</a>
+        </li>
+        {% endif %}
+        {% if current_user.has_permissions('manage_templates') and user_has_template_permission %}
+        {% if not template._template.archived %}
+        <li>
+          <a class="usa-link margin-right-3"
+          href="{{ url_for('.delete_service_template', service_id=current_service.id, template_id=template.id) }}">Delete this
+          template</a>
+        </li>
+        {% endif %}
+        {% if not template._template.redact_personalisation %}
+        <li>
+          <a class="usa-link" href="{{ url_for('.confirm_redact_template', service_id=current_service.id, template_id=template.id) }}">Hide all personalized and conditional content after sending</a> for increased privacy protection
+        </li>
```

Comment:
![image](https://github.com/GSA/notifications-admin/assets/53159604/b9cafce4-ffcb-4e86-a7ba-1e35f9ae5356)

this is how it looks for me.

Reply:
@jonathanbobel has this been addressed yet, or is it intentional to keep things this way?

Reply:
@heyitsmebev hey I don't see the same bulleted list because I have the `<ul class="usa-list usa-list--unstyled">` applied, so not sure why there would be the discrepancy.

Reply:
Was `make bootstrap` run after pulling down a local copy of this PR?  That might be why it didn't look right, if the assets weren't rebuilt.

Since all else looks okay and it should be accounted for given the style change, let's see how it looks on staging to verify.  We can always fix/adjust further if need be!